lib/domains/character/CharacterType.tsx

```diff
@@ -141,6 +143,13 @@ export const CharacterTemplatesInformation: {
   [CharacterTemplates.TheWitchIsDead]: {
     author: { name: "Grant Howitt", link: "https://rowanrookanddecard.com/" },
   },
+  [CharacterTemplates.EdgeOfTheEmpire]: {
+    author: {
+      name: "Fantasy Flight",
+      link:
+        "https://www.fantasyflightgames.com/en/products/star-wars-edge-of-the-empire/",
+    },
+  },
```

Comment:
**fyi** @christiefelker I think in the future we will be able to use the Link block type to get rid of those hardcoded links in the code. What do you think ?

(unrelated to your code @Gpoitras)

Reply:
Yeah this seems like a good use for it : )